platform-core-module/src/main/java/com/softicar/platform/core/module/file/smb/testing/SmbTestServerConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package com.softicar.platform.core.module.file.smb.testing;
+
+public class SmbTestServerConfiguration {
+
+	private final String testPassword;
+	private final String testShare;
+	private final String testUser;
+
+	public SmbTestServerConfiguration(String testPassword, String testShare, String testUser) {
```

Comment:
the order of the 3 parameters is strange, usually it is, 1. host 2. user 3. pass

Reply:
I agree. Not sure how alphabetical order ended up winning here.

Reply:
Actually, after thinking about it, I would recommend to drop that c'tor and add setter methods with chaining. See new class Locale implementing ILocale.

Reply:
This whole class was discarded.